src/applications/find-forms/containers/SearchResults.jsx

```diff
@@ -31,11 +33,26 @@ export class SearchResults extends Component {
     updatePagination: PropTypes.func.isRequired,
   };
 
-  componentDidUpdate(previousProps) {
-    const justRefreshed = previousProps.fetching && !this.props.fetching;
+  constructor(props) {
+    super(props);
 
-    if (justRefreshed) {
+    this.state = { howToSort: '', sortedResults: [] };
+    this.sortOptions = ['Last Updated (Newest)', 'Last Updated (Oldest)'];
+  }
+
+  componentDidMount() {
+    const { props, updateSortedResultsState } = this;
+    if (props.results && props.results.length > 0) updateSortedResultsState();
```

Comment:
We should probably have a linting rule about this that says that you either always have to include `{}` for if statements or if it's just a single line then you can drop them. Not opinionated personally but I've noticed this repo tends to favor the former I think when I've looked throughout our code 🙂

Reply:
Yeah. We probably should have a linter rule.